client/src/pages/Profile.js

```diff
@@ -384,21 +390,33 @@ const Profile = ({
                 />
               )}
               {!ownUser && (
+                <>
                 <MessageModal
                   isAuthenticated={isAuthenticated}
                   isFromUserCard={"USER"}
                   isFromProfile={true}
                   postAuthorName={`${firstName} ${lastName}`}
                   authorId={userId}
                 />
+                {/* <ThankButton>Thanks</ThankButton> */}
+
+                </>
               )}
             </NameDiv>
             {about && <DescriptionDesktop> {about} </DescriptionDesktop>}
             <IconsContainer>
-              <HelpContainer>
+              <MobileLocation>
```

Comment:
did you confirm with cami regarding location of address on mobile and desktop?

Reply:
yes, confirmed